bindings/node.js/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "devDependencies": {
+    "@types/node": "^14.14.14",
+    "typescript": "^4.0.3"
```

Comment:
I'm prefer-actual-minimal-requirement-instead-of-whatever-latest-for-the-moment kind of guy, and I had no problem with 3.8.3. In other words "^3" would supposedly do. However...

I'm also trying to keep things at bare minimum on my desktop, and when I tried the `npm install` as you suggested, it resulted in errors. After which I've figured that if I start from scratch and go straight for `tsc`, the one I have on `$PATH`, I get a meaningful message. It suggested to run `npm i @types/node`, after which it succeeded. And it doesn't matter if I have package.json **or not**. In other words it appears that at least for intended purpose (of testing things prior pinging blst-ts for feedback and integration) this file appears to be superfluous. Well, it doesn't prevent me from running `npm install --global typescript@3` or `npm install @types/node@10`...

Reply:
It's not strictly necessary now, but if you want to distribute this package through NPM you are required to have a package.json.